index.js

```diff
@@ -1,10 +1,14 @@
-import {requireNativeComponent} from 'react-native';
+import { requireNativeComponent } from "react-native";
 
-export {default as VizbeeManager} from './VizbeeManager';
-export {default as VizbeeSignInDelegate} from './VizbeeSignInDelegate';
-export {default as VizbeeSignInInfo} from './VizbeeSignInInfo';
-export {default as VizbeeVideo} from './VizbeeVideo';
-export {default as VizbeeVideoStatus, VizbeePlayerState} from './VizbeeVideoStatus';
-export {default as VizbeeTrackStatus} from './VizbeeTrackStatus';
-export {default as VizbeeTrackInfo} from './VizbeeTrackInfo';
-export const VizbeeCastButton = requireNativeComponent('VizbeeCastButtonView');
\ No newline at end of file
+export { default as VizbeeManager } from "./VizbeeManager";
+export { default as VizbeeSignInDelegate } from "./VizbeeSignInDelegate";
+export { default as VizbeeSignInInfo } from "./VizbeeSignInInfo";
+export { default as VizbeeVideo } from "./VizbeeVideo";
+export {
+  default as VizbeeVideoStatus,
+  VizbeePlayerState,
+} from "./VizbeeVideoStatus";
+export { default as VizbeeTrackStatus } from "./VizbeeTrackStatus";
+export { default as VizbeeTrackInfo } from "./VizbeeTrackInfo";
+export const VizbeeCastButton = requireNativeComponent("VizbeeCastButtonView");
+export { default as VizbeeMiniCastBar } from "./src/mini-cast-bar";
```

Comment:
let's move it to the "components" folder, something like this "./src/components/mini-cast-bar"

Reply:
Also, The convention widely adopted is PascalCase (UpperCamelCase) for component names and folder structures. Can you recheck on this rename to `VizbeeMiniCastBar`?

Reply:
I see that now it's exported as `export { VizbeeMiniCastBar }` which enforces using the specified name when other modules want to import it. I think we should give flexibility to other modules in choosing a name during the import